### PR TITLE
ATB-1499 | Store Dashboard in ais-infra repo 

### PR DESCRIPTION
**Pull request by MinaMahdi**
### What? 

Storing our AIS Dynatrace dashboard 

### Why? 

Keeping a copy in our repo in case any changes are made to it

### CODE REVIEW

File: ais-dashboard/account_interventions_service.json [hunk 1]
```diff
@@ -0,0 +1,4058 @@
+{
+  "metadata": {
+    "configurationVersions": [
+      7
+    ],
+    "clusterVersion": "1.284.110.20240210-104704"
+  },
+  "id": "8730e683-d596-49d4-bd9c-2f66ee886b17",
+  "dashboardMetadata": {
+    "name": "[WIP] Accounts Intervention Service",
```
Comment (domantas-gds) [inline ais-dashboard/account_interventions_service.json]: Is the dashboard still a work in progress or can we remove `[WIP]` from the name?

File: ais-dashboard/account_interventions_service.json [hunk 2]
```diff
@@ -0,0 +1,4058 @@
...
+    "shared": true,
+    "owner": "taher.kapasi@digital.cabinet-office.gov.uk",
+    "dashboardFilter": {
+      "managementZone": {
+        "id": "902899677339926396",
+        "name": "di-id-reuse-core-build"
```
Comment (domantas-gds) [inline ais-dashboard/account_interventions_service.json:16]: build specific?

File: ais-dashboard/account_interventions_service.json (hunk 1 above)
Comment (MinaMahdi) [reply ais-dashboard/account_interventions_service.json]: Yeah I'm going to remove WIP, I think we should be happy with it 👍

File: ais-dashboard/account_interventions_service.json (hunk 2 above)
Comment (MinaMahdi) [reply ais-dashboard/account_interventions_service.json:16]: Shall I change to All in the management zone  instead, rather than a specific env?

Comment (domantas-gds) [reply ais-dashboard/account_interventions_service.json:16]: Not major but also no harm having it more generic in our repo :) if you're already updating to remove [WIP] prefix

**Review by domantas-gds — APPROVED** [review]

**Review by TaherKapasi — APPROVED** [review]

**Pull request merged**